### DISCUSSION TITLE
Invalid Mortyhash Parameter

### DISCUSSION THREAD

**Discussion by vbuser2004** [searx/searx#2964] [Installation issue: Official wiki]
Version: 1.0.0 with nginx 
Cloned from this repo.

**How did you install Searx?**
Installed using official scripts on Ubuntu 21.04

**What happened?**
Created Morty base64 key and added to settings.yml file (using command: openssl rand -base64 33). 

When attempting to get proxied results, get error:

```
Error: invalid "mortyhash" parameter
Warning! This instance does not support direct URL opening.
```

I confirmed in the morty.service file that the -key is the same as the one in settings.yml
Morty log shows no errors (except invalid hash parameter). 

**How To Reproduce**
Install using scripts.  

**Expected behavior**
Proxified results should open.

**Comment by acaN** [comment]
For some reason including -key 'mykey' in morty command line parameters doesn't work.

Workaround is to set key as an environment variable using MORTY_KEY.
If you followed wiki steps, you can modify morty.service file to include it in "Environment" parameter:
`Environment=USER=morty HOME=/usr/local/morty DEBUG=true MORTY_KEY='mykey'`
